kovri.i2p/_i18n/en.yml

```diff
@@ -67,8 +67,8 @@ get-started:
   opsec_list2-3: with RBAC)
   opsec_list3-1: After installing the appropriate resources in your kovri data path, consider setting appropriate access control with
   opsec_list3-2: ", or whatever your OS uses for ACL."
-  opsec_list3-3: "Note: see kovri.conf to find your data path for Linux/OSX/Windows"
-  opsec_list4: Never share your port number with anyone as it will effect your anonymity!
+  opsec_list3-3: "Note: see kovri.conf to find your data path for Linux/macOS/Windows"
```

Comment:
The macOS classification wasn't introduced until 10.12. We support 10.11 as well. Technically, this isn't necessarily a typo.